### ISSUE TITLE
A11y: Elements use prohibited ARIA attributes - Image Gallery Slider

### ISSUE THREAD

**Issue by w3work** [shopware/platform#7080]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.10.1  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The Image gallery slider inside the Productdetail-Page gets the following error message in Lighthouse:
**Elements use prohibited ARIA attributes**
Using ARIA attributes in roles where they are prohibited can mean that important information is not communicated to users of assistive technologies.

Failing element:
<div class="gallery-slider-thumbnails-item tns-nav-active tns-item tns-slide-active" data-nav="0" tabindex="-1" aria-label="Carousel Page 1 (Current Slide)" aria-controls="tns1" id="tns2-item0">

![Image](https://github.com/user-attachments/assets/07a88901-db75-4399-acff-e722085e4093)

*_Expected behaviour:_*

Dont know, what the correct expected behaviour should be. Seems to be the role is not correct.

*_How to reproduce:_*

Can be reproduced in a clean new Shopware Shop, with an Image gallery in Prduct detail page

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40837**



[created from NEXT-40837, comment 541741]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40837, comment 542128]

**Comment by Phil23** [comment]
Hey @w3work , thank you for reporting this issue.

The aria attributes are set by a third-party plugin called "tiny slider" that we use for the image gallery. We have already made several patches to this plugin, but it is not maintained properly anymore, so we decided not to invest further in it. 

Our goal is to devise a new solution for the sliders in the image gallery. Sliders present a special accessibility challenge. Therefore, we would like to invest in a new component that better reflects all the best practices. For now, there won't be any changes to the current plugin.

Thx for your patience.
